config/locales/models/report.en.yml

```diff
@@ -33,15 +33,25 @@ en:
         comment: Comment
   tabs:
     report:
-      summary: Summary
-      activities: Activities
-      forecasts: Forecasts
-      actuals: Actuals
-      variance: Variance
-      budgets: Budgets
-      current: Current
-      approved: Approved
-      comments: Comments
+      summary:
+        heading: Summary
+      activities:
+        heading: Activities
+      forecasts:
+        heading: Forecasts
+      actuals:
+        heading: Actuals
+      variance:
+        heading: Variance
+        body: This tab shows the variance between forecasted and actual spend, including any adjustments, in this reporting cycle.
```

Comment:
I suspect that the change in behaviour contained here would be much clearer if you broke this down into 2 commits:

1. refactor with no changes to content
2. change in content to add the variance body "This tab shows the..."